packages/esbuild/test/e2e/simple-case/build.js

```diff
@@ -10,25 +11,21 @@ module.exports.cssBundleProd = (build, options) => {
 };
 
 function run(cssInjection, build, options) {
+    const outdir = cssInjection === 'css' ? 'dist-bundle' : 'dist';
```

Comment:
maybe change the out-dir name to reflect the setting that it relates to? something like `dist-css/css-in-js`?

Reply:
I don't think it's necessary the test is already overloaded with terms so `css-in-js` might not be better than `dev`.